labelspark/__init__.py

```diff
@@ -72,18 +105,24 @@ def flatten_bronze_table(bronze_table):
 
 # processes bronze table into silver table
 def bronze_to_silver(bronze_table):
-    # valid_schemas = list(spark_schema_to_string(bronze_table.schema.jsonValue()))
     bronze_table = flatten_bronze_table(bronze_table)
-
     bronze_table = bronze_table.withColumnRenamed("DataRow ID", "DataRowID")
+
+    # video labels need special handling
+    video = False
+    if "Label.frameNumber" in bronze_table.columns:
```

Comment:
about what `video = Label.frameNumber" in bronze_table.columns`

Reply:
Also I would advise setting this up via an enum

```python
class AssetType(enum.Enum):
    Video = 1
    Text = 2 
   ....

def get_asset_type(d: DataRow) -> AssetType
```

Reply:
This is a bit beyond my current understanding of Python, so maybe we can enhance this for the next version after ship? The current check for "Label.frameNumber" is a pretty simple one that we can adjust later.